core/json/json.pest

```diff
@@ -0,0 +1,129 @@
+// see https://spec.json5.org/#syntactic-grammar and
+// https://spec.json5.org/#lexical-grammar
+
+COMMENT = _{ "/*" ~ (!"*/" ~ ANY)* ~ "*/" | "//" ~ (!line_terminator ~ ANY)* }
```

Comment:
Was this grammar hand-written based on the above spec or sourced from somewhere else?

Reply:
nvm, this file was moved instead of created here, so you might not know